airflow/configuration.py

```diff
@@ -330,13 +346,36 @@ def __init__(self, defaults, *args, **kwargs):
 
     def _validate(self):
         if self.get("core", "executor") != 'SequentialExecutor' \
-                and "sqlite" in self.get('core', 'sql_alchemy_conn'):
+                and "sqlite" in self.get_with_fallback('core', 'sql_alchemy_conn'):
             raise AirflowConfigException("error: cannot use sqlite with the {}".
                                                        format(self.get('core', 'executor')))
 
         self.is_validated = True
 
     def get(self, section, key, **kwargs):
+        return self._get(section, key, True)
+
+    def get_with_fallback(self, section, key, **kwargs):
+        ret_val = self._get(section, key, False)
```

Comment:
yes default will take precedence over the command. is this a note or a comment to change something?